README.md

```diff
@@ -90,8 +89,16 @@ release.
 #### Container Image
 
 The latest container image can be found at:
-* `registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.10.0` (arch: `amd64`, `arm`, `arm64`, `ppc64le` and `s390x`)
-* View all multi-architecture images at [here](https://explore.ggcr.dev/?image=registry.k8s.io%2Fkube-state-metrics%2Fkube-state-metrics:v2.10.0)
+<<<<<<< HEAD
```

Comment:
This seems to look like the rebase on main didn't work

Reply:
Thank you for pointing out it.
I've fixed the conflict.